pkgs/by-name/ka/karakeep/package.nix

```
@@ -81,56 +81,56 @@ stdenv.mkDerivation (finalAttrs: {
+    # Copy necessary files into lib/karakeep while keeping the directory structure
     LIB_TO_COPY="node_modules apps/web/.next/standalone apps/cli/dist apps/workers packages/db packages/shared packages/trpc"
-    HOARDER_LIB_PATH="$out/lib/hoarder"
+    KARAKEEP_LIB_PATH="$out/lib/karakeep"
     for DIR in $LIB_TO_COPY; do
-      mkdir -p "$HOARDER_LIB_PATH/$DIR"
-      cp -a $DIR/{.,}* "$HOARDER_LIB_PATH/$DIR"
-      chmod -R u+w "$HOARDER_LIB_PATH/$DIR"
+      mkdir -p "$KARAKEEP_LIB_PATH/$DIR"
+      cp -a $DIR/{.,}* "$KARAKEEP_LIB_PATH/$DIR"
+      chmod -R u+w "$KARAKEEP_LIB_PATH/$DIR"
     done
 
     # NextJS requires static files are copied in a specific way
     # https://nextjs.org/docs/pages/api-reference/config/next-config-js/output#automatically-copying-traced-files
-    cp -r ./apps/web/public "$HOARDER_LIB_PATH/apps/web/.next/standalone/apps/web/"
-    cp -r ./apps/web/.next/static "$HOARDER_LIB_PATH/apps/web/.next/standalone/apps/web/.next/"
+    cp -r ./apps/web/public "$KARAKEEP_LIB_PATH/apps/web/.next/standalone/apps/web/"
+    cp -r ./apps/web/.next/static "$KARAKEEP_LIB_PATH/apps/web/.next/standalone/apps/web/.next/"
 
     # Copy and patch helper scripts
     for HELPER_SCRIPT in ${./helpers}/*; do
       HELPER_SCRIPT_NAME="$(basename "$HELPER_SCRIPT")"
-      cp "$HELPER_SCRIPT" "$HOARDER_LIB_PATH/"
-      substituteInPlace "$HOARDER_LIB_PATH/$HELPER_SCRIPT_NAME" \
-        --replace-warn "HOARDER_LIB_PATH=" "HOARDER_LIB_PATH=$HOARDER_LIB_PATH" \
+      cp "$HELPER_SCRIPT" "$KARAKEEP_LIB_PATH/"
+      substituteInPlace "$KARAKEEP_LIB_PATH/$HELPER_SCRIPT_NAME" \
+        --replace-warn "KARAKEEP_LIB_PATH=" "KARAKEEP_LIB_PATH=$KARAKEEP_LIB_PATH" \
         --replace-warn "RELEASE=" "RELEASE=${finalAttrs.version}" \
         --replace-warn "NODEJS=" "NODEJS=${nodejs}"
-      chmod +x "$HOARDER_LIB_PATH/$HELPER_SCRIPT_NAME"
-      patchShebangs "$HOARDER_LIB_PATH/$HELPER_SCRIPT_NAME"
+      chmod +x "$KARAKEEP_LIB_PATH/$HELPER_SCRIPT_NAME"
+      patchShebangs "$KARAKEEP_LIB_PATH/$HELPER_SCRIPT_NAME"
     done
 
     # The cli should be in bin/
     mkdir -p $out/bin
-    mv "$HOARDER_LIB_PATH/hoarder-cli" $out/bin/
+    mv "$KARAKEEP_LIB_PATH/karakeep" $out/bin/
 
     runHook postInstall
   '';
 
   postFixup = ''
     # Remove large dependencies that are not necessary during runtime
-    rm -rf $out/lib/hoarder/node_modules/{@next,next,@swc,react-native,monaco-editor,faker,@typescript-eslint,@microsoft,@typescript-eslint,pdfjs-dist}
+    rm -rf $out/lib/karakeep/node_modules/{@next,next,@swc,react-native,monaco-editor,faker,@typescript-eslint,@microsoft,@typescript-eslint,pdfjs-dist}
 
     # Remove broken symlinks
     find $out -type l ! -exec test -e {} \; -delete
   '';
 
   meta = {
-    homepage = "https://github.com/hoarder-app/hoarder";
+    homepage = "https://karakeep.app/";
     description = "Self-hostable bookmark-everything app with a touch of AI for the data hoarders out there";
```

Maybe update the description from the Karakeep GitHub?

```suggestion
    description = "Self-hostable bookmark-everything app (links, notes and images) with AI-based automatic tagging and full text search";
```